app/models/game.rb

```diff
@@ -1,17 +1,21 @@
 class Game < ActiveRecord::Base
   belongs_to :user
   belongs_to :sport
-  belongs_to :state
   has_many :players, dependent: :destroy
 
   validates :user, presence: true
-  validates :description, presence: true,
-    uniqueness: true,
-    length: { maximum: 20}
+  validates :description, uniqueness: true,
+    length: { maximum: 30}
   validates :sport, presence: true
   validates :street_address, presence: true
-  validates :city, presence: true
   validates :start_date, presence: true
   validates :max_players, presence: true,
     numericality: { only_integer: true }
+
+  include PgSearch
+  pg_search_scope :search,
+    :against => :street_address,
+    :using => {
+      :tsearch => {:prefix => true}
```

Comment:
Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.